process/sync/storageBootstrap/baseStorageBootstrapper.go

```diff
@@ -60,6 +60,12 @@ func (st *storageBootstrapper) loadBlocks() error {
 		if err != nil {
 			break
 		}
+
+		if round == headerInfo.LastRound {
+			err = sync.ErrCorruptBootstrapFromStorageDb
```

Comment:
errors are shadowed by line 100, add log.Warn for each error.

Reply:
done.